primitives/runtime/src/lib.rs

```diff
@@ -257,6 +262,72 @@ pub fn storage_value_key(pallet_prefix: &str, value_name: &str) -> StorageKey {
 	StorageKey(final_key)
 }
 
+#[derive(Encode, Decode, TypeInfo, PalletError)]
+pub enum OwnedBridgeModuleError {
+	/// All pallet operations are halted.
+	Halted,
+}
+
+pub trait OwnedBridgeModule<T: frame_system::Config> {
+	const NAME: &'static str;
```

Comment:
Let's, please, rename this to something like `LOG_TARGET` or something like that. Otherwise it is unclear what it actually does :)

Reply:
Done.